adapters/tlsio_ssl_esp8266.c

```diff
@@ -313,10 +314,11 @@ LOCAL int openssl_thread_LWIP_CONNECTION(TLS_IO_INSTANCE* p)
                                                 LogError("SSL_set_fd failed");
                                             }
                                             else{
-                                                LogInfo("SSL connect... ");
                                                 int retry_connect = 0;
                                                 int connect_succeeded = false;
 
+                                                LogInfo("SSL connect... ");
```

Comment:
>LogInfo("SSL connect... "); [](start = 48, length = 27)

Ignore this comment, but I hate random LogInfo that don't help anything.  If this happened to be removed it wouldn't hurt my feelings.